test/http.ts

```diff
@@ -309,19 +309,23 @@ test('DNS IPv4', withServer, async (t, server, got) => {
 	t.true(isIPv4(response.body));
 });
 
-if (IPv6supported) {
-	test('DNS IPv6', withServer, async (t, server, got) => {
-		server.get('/ok', echoIp);
+// Travis CI Ubuntu Focal VM does not resolve IPv6 hostnames
+testIPv6('DNS IPv6', withServer, async (t, server, got) => {
+	server.get('/ok', echoIp);
 
+	try {
 		const response = await got('ok', {
 			dnsLookupIpVersion: 'ipv6'
 		});
 
 		t.true(isIPv6(response.body));
-	});
-}
+	} catch (error) {
+		// If network unreachable (happens on TravisCI with Ubuntu Bionic and Focal)
+		t.is(error.code, 'ENETUNREACH');
+	}
```

Comment:
Use conditional `t.failing` instead. That way we'll know what tests should be fixed.

Reply:
I fail to understand how do you want to use `t.failing` can you make an example, thanks.

Reply:
@szmarczak meant `test.failing()`.

https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md#failing-tests

Reply:
Yes. Sorry for the confusion 😅

Reply:
Test test was already using a conditional `test`: `testIPv6`. I forgot to remove the `try, catch`. The thing is that it isn't failing...

Reply:
What about `test/stream.ts` and `test/unix-socket.ts`?